### ISSUE TITLE
bcm4343 support

### ISSUE THREAD

**Issue by nkrapivin** [seemoo-lab/nexmon#220]
My Galaxy Tab E SM-T561 seem to use bcm4343
gtel3g_defconfig
CONFIG_BT_BCM4343=y
CONFIG_BCM4343=y
CONFIG_WLAN_SDIO=y
CONFIG_WIFI_CONTROL_FUNC=y
CONFIG_BROADCOM_WIFI_RESERVED_MEM=y
CONFIG_BCMDHD_FW_PATH="/system/etc/wifi/bcmdhd_sta.bin"
CONFIG_BCMDHD_NVRAM_PATH="/system/etc/wifi/nvram_net.txt"
CONFIG_WIFI_BROADCOM_COB=y

Is bcm4343 supported?
OS: Android 4.4.4 TouchWiz
Kernel sources: available

**Comment by mirzaatifbaig** [comment]
I don't think it will Work on Android less than 6.0.This is because of how android handles code.Use lineage or other greater than cm 13 and use the app to install binaries and libs.Then get the few_bcmdhd.bin file from here rename and replace it in your firmware location.Set permissions and you are done.I think It's how the system handle binary data starting from android 6.0
